src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImportAliasCommand.cs

```diff
@@ -290,103 +378,118 @@ private Collection<AliasInfo> GetAliasesFromFile(bool isLiteralPath)
         {
             Collection<AliasInfo> result = new Collection<AliasInfo>();
 
+            // it seems like filePath is always null and never defined, so we can remove it as far as I could see
+            // bug keeping it for now because tests are failing
```

Comment:
No, see line 528.

Reply:
outdated comment, should have been removed